src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -1180,6 +1222,15 @@ def _deny_request(
                 "message": privacy_requests.reason,
             },
         )
+        if config.notifications.send_request_review_notification:
+            _send_privacy_request_review_email_to_user(
+                db=db,
+                action_type=EmailActionType.PRIVACY_REQUEST_REVIEW_DENY,
+                email=privacy_request.get_cached_identity_data().get(
+                    ProvidedIdentityType.email.value
+                ),
+                rejection_reason=privacy_requests.reason,
+            )
```

Comment:
There's some discrepancies in the order things happen in the approve/deny flows, some introduced here, some already existing.  We need to 1) update the privacy request, 2) create an audit log, and 3) potentially send an email.    I might refactor both this and `_approve_request` to follow this order, or at least put these steps in the same order.

Reply:
good idea, thanks!